### PR TITLE
Add MAC addresses page in Network menu

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready/Hold | [Ticket](https://pi-top.atlassian.net/browse/<ticket-number>) |

#### Main changes
-

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: tests/network_test.py [hunk 1]
```diff
@@ -85,3 +86,45 @@ def test_usb(miniscreen, snapshot, internal_ip):
...
+    miniscreen.down_button.release()
+    sleep(1)
+    miniscreen.down_button.release()
+    sleep(1)
+    miniscreen.down_button.release()
+    sleep(1)
+
+    snapshot.assert_match(miniscreen.device.display_image, "mac_addresses_no_ap.png")
+
+    # Display AP MAC address
```
Comment (jcapona) [inline tests/network_test.py:127]:
```suggestion
    # Display AP MAC
```

**Review by angusjfw — APPROVED** [review]
looks good but i have not run it
